engine/clients/redis/upload.py

```diff
@@ -20,13 +20,12 @@ def init_client(cls, host, distance, connection_params, upload_params):
     def upload_batch(
         cls, ids: List[int], vectors: List[list], metadata: Optional[List[dict]]
     ):
-
         p = cls.client.pipeline(transaction=False)
         for i in range(len(ids)):
             idx = ids[i]
             vec = vectors[i]
             meta = metadata[i] if metadata else {}
-            meta = meta or {}
+            meta = {k: v for k, v in meta.items() if v is not None} or {}
```

Comment:
`or {}` makes no sense

Reply:
Yeah, `meta or {}` didn't make sense as well, but I just filled the gaps :stuck_out_tongue: